### PR TITLE
Feature/homepage

### PR DESCRIPTION
**Pull request by WebsByTodd**

### CODE REVIEW

**Comment by nedtwigg** [comment]
image

Note the rearranged and slightly condensed icon layout in the second entry.  I have a weak visual preference for consolidating the "closed" icons along the bottom, and leaving the "open" icon with the open letters at top.  I also think that tucking the timestamp closer under the name helps with a gap in the right border.

**Comment by nedtwigg** [comment]
In the preview pane, it would be good for the two links to be more clearly links.  Perhaps a round border around them.

On mobile, it might help to make the buttons multi-line, e.g.

```
View full
context

Copy URL
to quote

```

image

**Comment by nedtwigg** [comment]
In building the infrastructure for a flexible social embed system, I could see that we were going to have a merge conflict.  I merged a pure refactor to `master`, and I'm about to fix the merge conflicts in this PR.  With this done, we shouldn't be able to work in parallel without further conflicts.

File: client/src/main/scripts/components/SearchBar.tsx [hunk 1]
```diff
@@ -33,18 +32,14 @@ class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     super(props);
 
     this.state = {
-      value: props.searchTerm,
+      value: props.searchTerm || "",
```
Comment (nedtwigg) [inline client/src/main/scripts/components/SearchBar.tsx:35]: This is a cool trick

**Comment by WebsByTodd** [comment]
I've created issues and project board items for your requests. Merging this now.

**Pull request merged**